src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The `href` attribute is pointing to a `.scss` file. SCSS files need to be compiled into CSS before they can be linked in an HTML document. Please ensure that the SCSS is compiled to a `.css` file and update the `href` to point to the compiled CSS file.